doc/release_notes.rst

```diff
@@ -34,6 +34,8 @@ E.g. if a new rule becomes available describe how to use it `snakemake -j1 run_t
 
 * Revise bus region definition by gadm. `PR #1001 <https://github.com/pypsa-meets-earth/pypsa-earth/pull/1001>`__
 
+* Minor bug-fixing for the generalised AC line types. `PR #1011 <https://github.com/pypsa-meets-earth/pypsa-earth/pull/1011>`__
+
```

Comment:
I wonder if we can possibly merge the release comment with `* Minor bug-fixing to get the generalised line types work for DC lines. `PR #1008 <https://github.com/pypsa-meets-earth/pypsa-earth/pull/1008>`__`, as both PRs do basically the same. What do you think?

We can take as an example:
```python
Minor bug-fixing for GADM_ID format naming. `PR #980
```